docs/developers-guide/drivers/plugins.md

```diff
@@ -9,6 +9,9 @@ info:
   name: Metabase SQLite Driver
   version: 1.0.0-SNAPSHOT-3.25.2
   description: Allows Metabase to connect to SQLite databases.
+contact-info:
+  name: Toucan McBird
+  info: toucan.mcbird@example.com
```

Comment:
looks to me that the spec and ts type definitions expect to see `contact-info.address` instead of `contact-info.info`. Is that right?

Reply:
Great catch, thanks! I've correct it.